plugins/woocommerce/client/blocks/assets/js/data/cart/selectors.ts

```diff
@@ -26,17 +27,21 @@ export const getCartData = ( state: CartState ): Cart => {
 	return state.cartData;
 };
 
-export const getCustomerData = (
-	state: CartState
-): {
-	shippingAddress: ShippingAddress;
-	billingAddress: BillingAddress;
-} => {
-	return {
-		shippingAddress: state.cartData.shippingAddress,
-		billingAddress: state.cartData.billingAddress,
-	};
-};
+export const getCustomerData = createSelector(
```

Comment:
Why is this the only selector updated to `createSelector` what is the benefit compared to leaving it as-is?

It looks like most selectors just select a single item directly from state, but `getCartItem`, `isItemPendingDelete`, and `isItemPendingQuantity` do some work before returning a value.

Reply:
Right but the others only return booleans or objects that already exist in state. Got it ✋🏼